src/scss/modifiers.scss

```diff
@@ -26,6 +26,10 @@ $spacing-map: (
     visibility: hidden;
   }
 
+  &--error {
+    color: $red;
+  }
+
```

Comment:
Replacement for `input-error` that only modifies the color, thus a modifier `mod--error` that will hopefully help reduce and eliminate many `global.scss` styles with BEM oriented ones